Sources/RLogger/RLogger.swift

```diff
@@ -31,7 +31,32 @@ public struct RLogger {
          default: ()
          }
          return message
-     }
+    }
+
+    @discardableResult
+    public static func verbose(message: String) -> String? {
```

Comment:
Could be more concise without requiring a named parameter (i.e. `verbose(_ message: String)`). Does your test compile? Your test seems to already exclude the named parameter 🤔

Reply:
@apparition47  I intentionally wanted a named parameter so there is no ambiguity with Variadicable functions.

This one is based on `@discardableResult static func verbose(_ format: String, arguments: CVarArg...) -> String?`:
RLogger.verbose("test")

Reply:
Ah I missed those function signatures, thanks